packages/api/src/command/hie/unlink-patient-from-organization.ts

```diff
@@ -161,10 +161,16 @@ function getDocumentsWithOid(
     const patient = document.contained?.find(isPatient);
     if (!patient) continue;
 
-    const identifier = patient.identifier?.find(identifier => identifier.system === urnOid);
+    const identifier = patient.identifier?.find(identifier => identifier.system?.includes(urnOid));
```

Comment:
Wasnt able to find a system in certain cases because it had some numbers appending

Reply:
Taken to Slack: https://metriport.slack.com/archives/C04CXLSAF7V/p1744412120024239?thread_ts=1744407461.576229&cid=C04CXLSAF7V